prime-router/docs/SFTP-SETUP.md

```diff
@@ -36,8 +36,11 @@ docker run -p 22:22 -v sftp:/home/foo/upload -d atmoz/sftp foo:pass:::upload
             host: localhost
             port: 2222
             filePath: ./upload
+            credentials: DEFAULT-FTP
```

Comment:
Instead of `DEFAULT-FTP` this should be called `DEFAULT-SFTP` so it's clear we're *not* using FTP.

Reply:
fixed